src/components/Dropdown/OptionsList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Option from './Option';
+import ClassList, { initialClass } from '../../utils/ClassList';
+
+const OptionsList = ({
+  dropdownOpen,
+  options,
+  onClick
+}) => {
+  const handleClick = value => () => onClick(value);
+
+  const getClassName = () => {
+    const classes = initialClass(['dropdown__options']);
+    const newClasses = ClassList.of([
+      dropdownOpen
+        ? 'dropdown__options--visible'
```

Comment:
but if we're already doing everything that way maybe not a good time to change it. Just an fyi maybe for the future.